gitlab-examples/gitlab-pipline/README.md

```diff
@@ -0,0 +1,23 @@
+# GitLab pipeline example
+This is an example of a GitLab pipeline which uses JFrog CLI to integrate with Artifactory.
+This sample uses a Gradle project, builds it, and pushes the artifact to Artifactory, including build info, environment variables, and SBOM.
```

Comment:
"...and SBOM." - which SBOM are you referring to?

Reply:
I mean Xray SBOM, as I am also scanning with Xray